src/solc_0.8/defi/ERC20RewardPool.sol

```diff
@@ -69,6 +69,7 @@ contract ERC20RewardPool is
         IERC20 rewardToken_,
         address trustedForwarder
     ) StakeTokenWrapper(stakeToken_) {
+        require(address(rewardToken_) != address(0), "ERC20RewardPool: zero address");
```

Comment:
should check that rewardToken_ is a contract

Reply:
done -> https://github.com/thesandboxgame/sandbox-smart-contracts/commit/82734c1f0cd79530586135733b7ad459e907d8d2